LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Coderetreat.org Contributors
+Copyright (c) 2019 Coderetreat.org Contributors
```

Comment:
Thanks @Erdbeervogel !! It's a small change, but it made me ask these questions:
- Is it necessary to make this change every year?
- If yes, can we do it automatically?

I found these answers:
- Yes, because we change the content: [answers in StackExchange](https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year)
- I found [this project](https://github.com/update/updater-license), but its execution should be scheduled. I was looking for some option at GitHub, like "Hi! You have a MIT Licence with a copyright from XXXX and you made changes. Would you like to update the year?" :sweat_smile: 

Finally, I agree this change right now :smile: Thanks!!

Reply:
@rachelcarmena I think I came across the same thoughts some time ago, but for some reason, we didn't proceed. Maybe it's good to automate it now.